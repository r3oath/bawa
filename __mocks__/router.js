import Router from 'next/router';

const mockedRouter = { push: () => {}, prefetch: () => {} };

Router.router = mockedRouter;
