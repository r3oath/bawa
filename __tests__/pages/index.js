import React from 'react';
import renderer from 'react-test-renderer';
import Index from '@app/pages/index';
import '@mocks/router';

describe('Index page', () => {
  it('Matches previous snapshot', () => {
    const component = renderer.create(<Index />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
