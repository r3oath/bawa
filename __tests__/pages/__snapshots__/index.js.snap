// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page Matches previous snapshot 1`] = `
<button
  className="button-l4um5n-0 cftsCa"
  onClick={[Function]}
>
  Next page
</button>
`;
